web-module/src/main/webapp/resources/structEditJS/autocomplete.mjs

```diff
@@ -160,10 +188,9 @@
         console.log("lookup exception:", e.statusText);
       }
     };
-  };
+  }
 
-  (document.readyState === 'loading')
-      ? document.addEventListener('DOMContentLoaded', checkInitAutocomplete)
-      : checkInitAutocomplete();
+}
+export const celAutocompleteInstance = new CelAutocompleteInitialiser();
+window.CELEMENTS.structEdit.autocomplete.autocompleteInstance = celAutocompleteInstance;
```

Comment:
```suggestion
export default const instance = new CelAutocompleteInitialiser();
window.CELEMENTS.structEdit.autocomplete.initaliserInstance = instance;
```

Reply:
this is not valid js. see https://stackoverflow.com/questions/36261225/why-is-export-default-const-invalid

Reply:
interesting, I wouldn't have guessed that!